data/localizations/categories/en.yml

```diff
@@ -31040,6 +31040,54 @@ en:
     vp-2-1:
       name: Aircraft
       context: Vehicles & Parts > Vehicles > Aircraft
+    vp-2-1-1:
+      name: Heavier-Than-Air Aircraft
+      context: Vehicles & Parts > Vehicles > Aircraft > Heavier-Than-Air Aircraft
```

Comment:
I'm not sure if I agree with having `Heavier-Than-Air Aircraft` as its own category 🤔 It seems unlikely that someone would ever pick it instead of one of the subcategories.

I think I would remove `Heavier-Than-Air Aircraft`and `Lighter-Than-Air Aircraft` and just move all the subcategories to be directly under `Aircraft`.